proxy/hdrs/HTTP.cc

```diff
@@ -625,7 +625,27 @@ http_hdr_type_set(HTTPHdrImpl *hh, HTTPType type)
 /*-------------------------------------------------------------------------
   -------------------------------------------------------------------------*/
 
-void
+bool
+is_version_supported(const uint8_t major, const uint8_t minor)
```

Comment:
Can you clarify what "supported" means? Having a (doxygen) comment for this function would be helpful.

Do you think we should not have 3(.0) here yet?

Reply:
Added a comment. I think we can update this when 3.0 is supported